devops/scripts/aad/create_api_application.sh

```diff
@@ -147,6 +147,8 @@ msGraphObjectId=$(az ad sp show --id ${msGraphAppId} --query "id" --output tsv -
 
 roleUserReadAll=$(get_msgraph_role "User.Read.All" )
 roleDirectoryReadAll=$(get_msgraph_role "Directory.Read.All" )
+groupsReadWriteAll=$(get_msgraph_role "Groups.ReadWrite.All" )
```

Comment:
Not sure this should be requried. Need to test without.

Reply:
When we create the groups using TF in the workspace:

```
resource "azuread_group" "workspace_owners" {
  count            = var.create_aad_groups ? 1 : 0
  display_name     = "${var.workspace_resource_name_suffix} Workspace Owners"
  owners           = [var.workspace_owner_object_id]
  security_enabled = true
}
```

We could add the TRE API as a group owner.